reflex/utils/prerequisites.py

```diff
@@ -1370,12 +1350,17 @@ def initialize_app(app_name: str, template: str | None = None):
         return
 
     # Get the available templates
-    templates = fetch_app_templates() or {}
-    if template is None:
-        template = prompt_for_template(templates)
+    templates: dict[str, Template] = fetch_app_templates() or {}
```

Comment:
nit: `fetch_app_templates` always return a dict, no need for `or {}`

Reply:
good catch, fixed